post-processing/post_processing.py

```diff
@@ -62,64 +67,15 @@ def run_post_processing(self, config: ConfigHandler):
             print(self.df.to_json(orient="columns", indent=2))
 
-        df = pd.DataFrame()
-        df_csv_export = pd.DataFrame()
-        # put all perflog information in one dataframe
-        for file in log_files:
-            try:
-                temp = read_perflog(file)
-                df = pd.concat([df, temp], ignore_index=True)
-            except KeyError as e:
-                if self.debug:
-                    print("Discarding %s:" %os.path.basename(file),
-                          type(e).__name__ + ":", e.args[0], e.args[1])
-                    print("")
-        if df.empty:
-            raise FileNotFoundError(errno.ENOENT, "Could not find a valid perflog in path", log_path)
-        # specify columns to export from dataframe to csv
-        if config.get("csv_export") is None:
-            raise KeyError("Missing csv_export (specify an empty list [] if none are required).")
-        else:
-            for col in config["csv_export"]:
-                df_csv_export = pd.concat([df_csv_export, df[col]], axis=1, join='outer')
-                df_csv_export.to_csv(log_path+'/output.csv', index=False)  # Set index=False to exclude the DataFrame index from the CSV
-            if self.debug:
-                print("Selected dataframe to export CSV file:")
-                print(df_csv_export)
-        # get axis columns
-        columns = [config["x_axis"]["value"], config["y_axis"]["value"]]
-        if config["x_axis"]["units"].get("column"):
-            columns.insert(1, config["x_axis"]["units"]["column"])
-        if config["y_axis"]["units"].get("column"):
-            columns.append(config["y_axis"]["units"]["column"])
-
-        series = config["series"]
-        # extract series columns and filters
-        series_columns = [s[0] for s in series]
-        series_filters = [[s[0], "==", s[1]] for s in series]
-        # check acceptable number of series
-        if len(set(series_columns)) > 1:
-            raise RuntimeError("Currently supporting grouping of series by only one column. \
-                               Please use a single column name in your series configuration.")
-        # add series columns to dataframe column list
-        for c in series_columns:
-            if c not in columns:
-                columns.append(c)
-
-        and_filters = config["filters"]["and"]
-        or_filters = config["filters"]["or"]
-        # extract filter columns
-        filter_columns = [f[0] for f in and_filters] + [f[0] for f in or_filters]
-
-        # FIXME: add scaling for x-axis
-        scaling_columns = []
-        # extract scaling columns
-        if config["y_axis"].get("scaling"):
-            if config["y_axis"]["scaling"].get("column"):
-                scaling_columns.append(config["y_axis"]["scaling"]["column"]["name"])
-
-        # gather all relevant columns
-        all_columns = set(columns + filter_columns + scaling_columns)
+        return self.df[config.plot_columns][self.mask]

```

Comment:
I assume this mess here is leftover from the merging with the refactored version?